docs/src/2.4.ManagingCamera.mdx

```diff
@@ -0,0 +1,170 @@
+
+```js
+// src/utils/useRequestAnimationFrame.js
+import { useEffect, useRef } from "react";
+
+export default function useRequestAnimationFrame(callback) {
+  const callbackRef = useRef();
+  useEffect(() => {
+    callbackRef.current = callback;
+  });
+
+  useEffect(() => {
+    const stop = window.requestAnimationFrame(() => {
+      callbackRef.current();
+    });
+
+    return function cleanup() {
+      cancelAnimationFrame(stop);
+    };
+  });
+}
+```
+
+Lastly, let's hook everything together:
+
+```js
+// src/Example.js
+
+// ...other imports
+
+import Duck from "./Duck";
+import useRequestAnimationFrame from "./utils/useRequestAnimationFrame";
+
+export default function Example() {
+  const cameraMoveSpeed = 1 / 100;
+  // use count to derive camera phi and thetaOffset values
+  const [count, setCount] = useState(0);
+  useRequestAnimationFrame(() => {
+    // update count before each browser repaint
+    const newCount = count + 2 >= steps ? 0 : count + 1;
+    setCount(newCount);
+  });
+
+  // generate cameraState from `count`
+  const cameraState = {
+    ...DEFAULT_CAMERA_STATE,
+    phi: count * cameraMoveSpeed * Math.PI,
+    thetaOffset: count * cameraMoveSpeed * 2 * Math.PI,
+  };
+
+  // ...other existing code
+
+  return (
+    <Worldview cameraState={cameraState}>
+      <Spheres>{[marker]}</Spheres>
+      <Axes />
+      <Duck />
+    </Worldview>
+  );
+}
+```
+
+Now the camera is moving around the scene, giving us 360° view of the duck model!
+
+_Notice we've added `cameraState={cameraState}`, which turns Worldview into a [controlled](https://reactjs.org/docs/forms.html) component. Controlled components usually require an `onChange` handler, but no `onCameraStateChange` prop is needed for our example, since we don't need to know the new internal `cameraState` value._
+
+<CodeSandboxEmbed src="https://codesandbox.io/s/qx5zrko8nj" />
+
+## Follow an Object
+
+In this step, let's move the duck along our knot, and move the camera to follow the duck around as it travels.
+
+Let's first modify the Duck component to accept children, so that the duck's position can be updated dynamically:
+
+```js
+// src/Duck.js
+const DEFAULT_CHILDREN = {
+  pose: {
+    position: { x: 0, y: 0, z: 0 },
+    orientation: { x: 0, y: 0, z: 0, w: 1 },
+  },
+  scale: { x: 3, y: 3, z: 3 },
+};
+
+export default function Example({ children = DEFAULT_CHILDREN }) {
+  return <GLTFScene model={duckModel}>{children}</GLTFScene>;
+}
+```
+
+Next, let's remove the camera setting for `thetaOffset` and `phi`, so that we don't feel dizzy observing the scene while the duck is moving. We can derive the duck position from the spheres over time, and use the duck position as `target` input for `cameraState`.
```

Comment:
We could do a thorough audit to capitalize all "spheres" and "cubes" to emphasize that these are React components.